roles/os_firewall/tasks/firewall/iptables.yml

```diff
@@ -21,7 +21,7 @@
   ignore_errors: yes
 
 - name: Install iptables packages
-  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=latest"
```

Comment:
Instead of using latest, we would probably want to separate out the iptables and iptables-services installation into separate tasks. We'll probably also have to use repoquery/dnf repoquery to query the locally installed version of iptables and force iptables-services to match that version....

Either that, or would we have a better experience by just shelling out to the native package management tools for this installation (I'm not sure if there is a broken dependency on the iptables/iptables-services packages or if it is related to the underlying ansible yum package module causing the issue).